interfaces/apparmor/spec.go

```diff
@@ -79,6 +79,15 @@ func RegisterSnippetKey(key string) SnippetKey {
 	return snippetKey
 }
 
+// GetSnippetKey retrieves a key from the list of valid keys
+func GetSnippetKey(key string) (SnippetKey, bool) {
```

Comment:
I'm not sure we need this,  a way to get all the registered keys would be useful though for tests

Reply:
I'll remove this and add that one.

Reply:
Done.